tests/testthat/test-mosaic.R

```diff
@@ -377,8 +371,8 @@ test_that("mosaic.dt() returns an appropriately sized data.table", {
   expect_equal(dt$yLabel[[1]][[1]],c('binB_a','binB_b'))
   expect_equal(length(dt$value[[1]]),2)
   expect_equal(length(dt$value[[1]][[1]]),2)
-  statsTable <- statsTable(dt)
-  expect_equal(names(statsTable), c(c('oddsratio', 'relativerisk', 'orInterval', 'rrInterval', 'pvalue')))
+  statsTable <- attributes(dt)$statsTable
```

Comment:
should be changed to attributes(dt)$statsTable in the rest of the file as well?

Reply:
no. so for a bit i was considering having the new statsTable work differently to the old one, making a separate attribute for it so that i could do something custom when writing out the json. but then i figured out the asJSON magic you noticed in the veupathUtils PR and changed my mind. so this is a useless change